scripts/runFSharpLint.fsx

```diff
@@ -154,3 +157,32 @@ Fsdk
     )
     .UnwrapDefault()
 |> ignore<string>
+
+// check .fsx files in target dir
+let allFiles = Fsdk.Misc.GetAllFilesRecursively targetDir
+
+let fsxFiles =
+    allFiles |> Array.filter(fun filename -> filename.EndsWith ".fsx")
```

Comment:
@webwarrior-ws any reason why not use List instead of Array? Or rather, Seq

Reply:
Using seq when you can is always preferable because it makes the code more resilient (when performance is not the goal), because this will still work even if we change Fsdk.Misc.GetAllFilesRecursively to return List instead of Array in the future

Reply:
Changed to using functions from `Seq` module.